docs/Contribute_To_Frontend_Quick_Guide.md

```diff
@@ -0,0 +1,243 @@
+
+ * Ensure the ability to log in to the **adm** node as the root user:
+    * vagrant up adm
+    * vagrant ssh
+    * su -
+    * ls -al /vagrant
+    * The Vagrantfile should be in the directory listing 
+
+## On the Local machine, i.e., not the vagrant virtual machine.
+
+For the desciption that follows, it will be assumed that the Vagrant file and virtual machine information reside in the directory:
+``` 
+~/vagrant-projects/vhpc 
+```
+## Contributing to the IML Frontend
+### Clone the Frontend repo, aka, the [GUI](https://github.com/intel-hpdd/GUI) repo
+```
+cd ~/vagrant-projects/vhpc
+git clone git@github.com:intel-hpdd/GUI.git 
+```
+### Create a branch, always work on a branch.
+```
+cd GUI
+git checkout -b  my-fix
+```
+### Validate that the correct branch has been selected.
+```
+git branch
+```
+### Work on the branch.
+```
+Use VS Code and open ~/vagrant-projects/vhpc/GUI
+```
+### As an Example, make a small change to the **Configuration** dropdown menu.
+In VS Code, locate and edit the following file: **app-states.js**
+```
+GUI 
+  > source
+    > iml
+      > app
+        > app-states.js
+
+Search for "Servers"
+```
+On the line containing "Servers", change **Servers** to **My Servers**:
+
+Change this line:
+```
+<li><a tabindex="-1" id="server-conf-item" ui-sref="app.server({ resetState: true })">Servers</a></li>
+```
+To look like this line:
+```
+<li><a tabindex="-1" id="server-conf-item" ui-sref="app.server({ resetState: true })">My Servers</a></li>
+```
+Save the file: **app-states.js**
+
+## Install dependencies and build the code
+
+### Install yarn
+Before you start using Yarn, you'll first need to install it on your system. 
+* How to [Install yarn](https://yarnpkg.com/lang/en/docs/install/)
+
+
+If yarn is installed properly, Install the external library dependencies
+
+Type the command:
+```
+yarn install
+```
+Build the code and Pass all required tests listed in **pacakage.json**
+```
+yarn watch
+```
+
+The watch command will leave the code in a state that will "watch" for further edits. If desired, the "watch" may be stopped by hitting ctrl-c
+
+### In a seperate terminal, log into the **adm** node
+* vagrant ssh adm
+
+Log in as root
+* su -
+
+Go to the cloned **GUI** repo on the /vagrant mount point where the code edits were made.
+* cd /vagrant/GUI
+
+* **yarn link**
+
+```
+yarn link v0.27.5
+success Registered "@iml/gui".
+info You can now run `yarn link "@iml/gui"` in the projects where you want to use this module and it will be used instead.
+Done in 0.60s.
+
+```
+The "yarn link" provides the ability to replace the 'installed' GUI that came with the installed IML with 'this' GUI with code edits.
+
+### Stop the IML running services
+* chroma-config stop
+
+### Replace the **gui** that is running on the **adm** node with the newly cloned **gui**
+
+The gui modules currently reside at /usr/share/chroma-manager/ui-modules on the running system.
```

Comment:
See above out of date comment.

Reply:
Fixed to create symbolic link to /usr/lib/iml-manager/iml-gui.